ipatests/test_integration/test_sssd.py

```diff
@@ -148,3 +149,34 @@ def test_is_user_filtered(self, user):
             dp_req = ("Looking up [{0}] in data provider".format(
                 self.users[user]['name']))
             assert not dp_req.encode() in sssd_log
+
+    def test_extdom_group(self):
+        """ipa-extdom-extop plugin should allow @ in group name.
+
+        Test for : https://bugzilla.redhat.com/show_bug.cgi?id=1746951
+
+        If group contains @ in group name from AD, eg. abc@pqr@AD.DOMAIN
+        then it should fetch successfully on ipa-client.
+        """
+        client = self.clients[0]
+        hosts = [self.master, client]
+        ad_group = 'group@group@{0}'.format(self.ad.domain.name)
```

Comment:
Nitpick but could you a comment that this group gets automatically created by PR-CI when the AD machine is setup? It means its presence is a prerequisite for this test, and for people manually running the test (outside of PRCI) with their own AD machine they would need to add this specific group manually.

Reply:
Yes this group will be created automatically in PR-CI in AD setup.
Once PR gets merged i'll update in https://www.freeipa.org/page/V4/AD_configuration_for_testing

Reply:
Thanks, I see that @netoarmando has already updated the wiki.